conf/scalardb-cluster-custom-values-direct-kubernetes-mode.yaml

```diff
@@ -56,33 +56,21 @@ scalardbCluster:
       cpu: 2000m
       memory: 4Gi
 
-  tolerations:
-    - effect: NoSchedule
-      key: scalar-labs.com/dedicated-node
-      operator: Equal
-      value: scalardb-cluster
-
   affinity:
-    nodeAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        nodeSelectorTerms:
-          - matchExpressions:
-              - key: scalar-labs.com/dedicated-node
-                operator: In
-                values:
-                  - scalardb-cluster
     podAntiAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        - labelSelector:
-            matchExpressions:
-              - key: app.kubernetes.io/name
-                operator: In
-                values:
-                  - scalardb-cluster
-              - key: app.kubernetes.io/app
-                operator: In
-                values:
-                  - scalardb-cluster
-          topologyKey: kubernetes.io/hostname
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                    - scalardb-cluster
+                - key: app.kubernetes.io/app
+                  operator: In
+                  values:
+                    - scalardb-cluster
+            topologyKey: kubernetes.io/hostname
+          weight: 50
```

Comment:
We use this [`preferredDuringSchedulingIgnoredDuringExecution`](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#types-of-inter-pod-affinity-and-anti-affinity) for new recommendations (`make the pods spread across the worker nodes`). 

When we use `preferredDuringSchedulingIgnoredDuringExecution`, Kubernetes tries to spread pods across the worker node. However, if there are less number of worker nodes than the number of pods, Kubernetes deploys two or more pods on one worker node.